From 53c13478c34c44c99e35129156659edf16af4132 Mon Sep 17 00:00:00 2001
From: Aichun Shi <aichun.shi@intel.com>
Date: Thu, 8 Dec 2022 15:02:47 +0800
Subject: [PATCH] Enable Intel In Field Scan(IFS) as kernel module
Patch-mainline: Not yet, from openEuler
References: bsn#22
openEuler-commit: 53c13478c34c44c99e35129156659edf16af4132
Modified-by-SEL: No


Intel inclusion
category: feature
feature: Intel In Filed Scan(IFS)
bugzilla: https://gitee.com/openeuler/intel-kernel/issues/I651S7
CVE: N/A

Intel-SIG: openeuler_defconfig: Enable Intel In Field Scan(IFS) as kernel module

--------------------------------------------

Enable Intel In Field Scan(IFS) as kernel module in openeuler_defconfig.

Signed-off-by: Aichun Shi <aichun.shi@intel.com>
Signed-off-by: Guoqing Jiang <guoqing.jiang@suse.com>
---
 arch/x86/configs/openeuler_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/x86/configs/openeuler_defconfig b/arch/x86/configs/openeuler_defconfig
index b582e8c87587..9f4e4c344ec9 100644
--- a/arch/x86/configs/openeuler_defconfig
+++ b/arch/x86/configs/openeuler_defconfig
@@ -6504,6 +6504,7 @@ CONFIG_XEN_FRONT_PGDIR_SHBUF=m
 # CONFIG_GREYBUS is not set
 # CONFIG_STAGING is not set
 CONFIG_X86_PLATFORM_DEVICES=y
+CONFIG_INTEL_IFS=m
 CONFIG_ACPI_WMI=m
 CONFIG_WMI_BMOF=m
 # CONFIG_ALIENWARE_WMI is not set
-- 
2.33.0

